exist-core/src/main/java/org/exist/collections/triggers/DocumentTriggers.java

```diff
@@ -53,7 +53,9 @@
     private SAXTrigger last = null;
     
     private final List<DocumentTrigger> triggers;
-    
+
+    private boolean skipTriggers = false;
```

Comment:
Likewise... These `triggersSuspended` checks should not be used here. Such checks are already present at the call site in DBBroker.

Reply:
@adamretter I fail to see where these checks are already present. Got me a pointer?

Reply:
See for example `MutableCollection#removeXMLResource`:

```
boolean useTriggers = broker.isTriggersEnabled();
                    if (CollectionConfiguration.DEFAULT_COLLECTION_CONFIG_FILE_URI.equals(name)) {
                        // we remove a collection.xconf configuration file: tell the configuration manager to
                        // reload the configuration.
                        useTriggers = false;
                        final CollectionConfigurationManager confMgr = broker.getBrokerPool().getConfigurationManager();
                        if (confMgr != null) {
                            confMgr.invalidate(getURI(), broker.getBrokerPool());
                        }
                    }

                    final DocumentTriggers trigger = new DocumentTriggers(broker, transaction, null, this, useTriggers ? getConfiguration(broker) : null);
```

Reply:
ah `MutableCollection`, already stared at that, and slapped my fingers when opening the editor :)
Will have a deeper look, thanks

Reply:
Not sure I understand.